manual/moderncv_userguide.tex

```diff
@@ -910,7 +966,21 @@ \section{Implementation details}
 \label{section:implementationDetails}
 
 \subsection{Creating your own styles}
-\todox{Explain how to create styles and and how to recombine headers, footers, bodies etc.}
+Currently {\moderncv} ships with five different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle} and \texttt{fancy}. Internally, these styles are numbered 1, 2, 3, 4 and 5, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
```

Comment:
```suggestion
Currently {\moderncv} ships with six different styles: \texttt{casual} (default), \texttt{classic}, \texttt{banking}, \texttt{oldstyle}, \texttt{fancy} and \texttt{contemporary}. Internally, these styles are numbered 1, 2, 3, 4, 5 and 6, respectively. All styles except \texttt{oldstyle} and \texttt{fancy} possess additional option parameters which were described in section \ref{section:customization:stylesAndOptions}.
```

Reply:
Ah, yes, in the meantime we already have six cool styles... Forgot that passage here, thanks for mentioning it!
Is now resolved with [e0e9259](https://github.com/moderncv/moderncv/pull/184/commits/e0e9259201620d26f233190f0d366627be653b81)